ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -245,7 +245,9 @@ a database ( directory )
           scope.database = true;
         }
         dbTracker.debugLog("database");
-        dbEventFound = true;
+        if (scope.database) {
```

Comment:
Do you need a separate check? Can be done at the same place where scope.database is set to true